src/data/exchange.graphql

```diff
@@ -1,8 +1,3 @@
-"""
-Represents either an action or a potential failure
-"""
-union ActionOrFailureUnion = OrderRequiresAction | OrderWithMutationFailure
-
```

Comment:
@starsirius I am feeling a bit uneasy. I copied my newest schema from exchange to metaphysics after adding the `shippingRadius` field on the order. But somehow this seems to remove some other things that you had just added a week ago. Why is this not on exchange main?

Reply:
Sorry for the delay. Looks like this has been resolved. Let me know if I can still help!